### PR TITLE
Improve the FAQ page content/styles.

### PR DESCRIPTION
**Pull request by biocom**
- Adds scroll offset, so that on click there's some margin above the title:

https://user-images.githubusercontent.com/31534717/116560136-633dfb80-a901-11eb-936c-1af9029e160c.mp4

- Styled links

https://user-images.githubusercontent.com/31534717/116560016-443f6980-a901-11eb-948f-979673f19e21.mp4

- Styling of the ::selection (on text select)

https://user-images.githubusercontent.com/31534717/116560039-499cb400-a901-11eb-8036-6f798c1e4b72.mp4

- Misc styling and text corrections.

### CODE REVIEW

**Comment by W3stside** [comment]
nice, i think we should leave links underline not just on hover. as is usually expected

**Comment by biocom** [comment]
> nice, i think we should leave links underlined, not just on hover. as is usually expected

Yeah my take was, this is an index, so you could say it's implied that those are links. Found it overwhelming to have all these items underlined. 

In contrast, I left them underlined on the content area given there it needs to be evident.

File: src/custom/pages/Faq/index.tsx [hunk 1]
```diff
@@ -15,6 +15,37 @@ const Wrapper = styled.div`
   }
 
   ${Content} {
+    > div a {
+      text-decoration: none;
```
Comment (W3stside) [inline src/custom/pages/Faq/index.tsx:19]: i wouldn't do this, hard to tell they're links otherwise imo

**Comment by W3stside** [comment]
> > nice, i think we should leave links underlined, not just on hover. as is usually expected
> 
> Yeah my take was, this is an index, so you could say it's implied that those are links. Found it overwhelming to have all these items underlined.
> 
> In contrast, I left them underlined on the content area given there it needs to be evident.

you coould add an icon next to the topic headers, then leave underlines for the sub headers

**Comment by biocom** [comment]
Personally I think it's fine, but I think either way works. Will adjust as to your recommendation (adding back the underlines to the index anchor links).

**Review by anxolin — APPROVED** [review]

**Pull request merged**